core/src/main/java/org/infinispan/commands/VisitableCommand.java

```diff
@@ -48,15 +48,23 @@
    boolean ignoreCommandOnStatus(ComponentStatus status);
 
    /**
-    * @return {@code true} if the command needs to read the previous values of the keys it acts on.
+    * @return Nodes on which the command needs to read the previous values of the keys it acts on,
+    * or <code>null</code> if an invocation does not make sense.
```

Comment:
Oops, I read "nodes" and I thought the javadoc came from a method returning a collection of addresses or something like that :)
When the invocation doesn't make sense, shouldn't the method throw an `IllegalStateException`?